Sources/ChatManager.swift

```diff
@@ -247,3 +262,62 @@ func pathFriendlyVersion(of component: String) -> String {
     // TODO: When can percent encoding fail?
     return component.addingPercentEncoding(withAllowedCharacters: allowedCharacterSet) ?? component
 }
+
+//MARK: Beams
+private let pushNotifications: PushNotifications = PushNotifications.shared
+
+extension ChatManager {
+    /**
+     Register device token with PushNotifications service.
+
+     - Parameter deviceToken: A token that identifies the device to APNs.
+
+     - Precondition: `deviceToken` should not be nil.
```

Comment:
Isn't this already enforced by the compiler?

Reply:
I've removed unnecessary comment. Thanks!